### PR TITLE
doc: how to create a new release.

### PR DESCRIPTION
**Pull request by NickNaso**
This PR add the guide that explain how to create and publish a new release.

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

File: CREATING_A_RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+From the root folder of the repo launch the following command:
+
+```bash
+> changelog-maker --format=markdown
+```
+
+* Use the output generated by **changelog maker** to update the
+[CHANGELOG.md][] following the style used in publishing the previous release.
+
+* Add any new contributors to the "contributors" section in the
```
Comment (legendecas) [inline CREATING_A_RELEASE.md:40]: I didn't find steps to create git tags for the release. Should we document that the tag should be created along with package.json version updates by `npm version patch` or `npm version minor`?

Comment (NickNaso) [reply CREATING_A_RELEASE.md:40]: The tag will be created by GitHub while creating new release.

Comment (legendecas) [reply CREATING_A_RELEASE.md:40]: Thanks, SGTM.

**Review by legendecas — APPROVED** [review]

**Pull request merged**